pkgs/development/libraries/libstrophe/default.nix

```diff
@@ -1,35 +1,33 @@
-{ stdenv, fetchFromGitHub, automake, autoconf, libtool, openssl, expat, pkgconfig, check }:
+{ stdenv, fetchFromGitHub, autoreconfHook, libtool, openssl, expat, pkgconfig, check }:
 
 stdenv.mkDerivation rec {
   pname = "libstrophe";
-  version = "0.10.0";
+  version = "0.10.1";
 
   src = fetchFromGitHub {
     owner = "strophe";
     repo = pname;
     rev = version;
-    sha256 = "1hizw695fw0cy88h1dpl9pvniapml2zw9yvxck8xvxbqfz54jwja";
+    sha256 = "11d341avsfr0z4lq15cy5dkmff6qpy91wkgzdpfdy31l27pa1g79";
   };
 
-  nativeBuildInputs = [ automake autoconf pkgconfig libtool check ];
-  buildInputs = [ openssl expat ];
+  nativeBuildInputs = [ autoreconfHook pkgconfig ];
+  buildInputs = [ openssl expat libtool check ];
```

Comment:
I am not sure if lobtool and check belong into nativeBuildInputs. Are they required at runtime?

Reply:
No, you are correct. `libmesode` is a fork of `libstrophe`, but I've not had the chance to sync them up until now. However, as with `libmesode` I cannot investigate the darwin issues.

Reply:
No problem. Can you please mark it as broken on darwin? So I won't nag you about this on the next update.

Reply:
> No problem. Can you please mark it as broken on darwin? So I won't nag you about this on the next update.

All done :)